### ISSUE TITLE
--insecure is not working with docker image ID a7e35d3527d2: x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by cloudwitch** [ankitpokhrel/jira-cli#432]
**Describe the bug**
Using the latest Docker image with ID a7e35d3527d2, I am receiving error `✗ Unable to generate configuration: x509: certificate signed by unknown authority` when trying to run `jira init --insecure`.

**Please provide following details**
1. JiraCLI Version:
    ```
    ~ # jira version
(Version="v1.0.0", GitCommit="4f6e8232b98602ca08f2d5f4a604339d70a7888d", CommitDate="2022-07-03T18:57:12+00:00", GoVersion="go1.17.11", Compiler="gc", Platform="linux/amd64")
    ```
2. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
    ```
    on-premise: Jira Server 8.20
    ```
3. What operating system are you using? Also mention version.
    ```
    Ubuntu 22.04
    ```
4. What terminal are you using? Also mention version.
    ```
    Gnome Terminal 3.44.0
    ```

**To Reproduce**

1. run:
    ```
    docker run -it --rm \
      --network host \
      -e "JIRA_AUTH_TYPE:bearer" \
      -e "JIRA_API_TOKEN:mYtOkEn" \
      ghcr.io/ankitpokhrel/jira-cli:latest
    ```
Note: The `--network host` is required due to my company's VPN software.
2. Run `jira init --insecure`.
3. Fill in prompts.
4. See error `✗ Unable to generate configuration: x509: certificate signed by unknown authority`.

**Expected behavior**
Not to get the x509 error.

**Additional context**
My company runs their own CA, I've tried copying the `/usr/local/share/ca-certificates/` to `/usr/local/share/ca-certificates/` in the container, then running `apk --no-cache add ca-certificates && update-ca-certificates` via Dockerfile, then using that image new to `jira init` and I still get the same error with or without the `--insecure` flag.

**Comment by ankitpokhrel** [comment]
HI @cloudwitch, this was fixed with #413 and the fix is available in the main branch. Could you please try with the build from main branch?

```sh
docker pull ghcr.io/ankitpokhrel/jira-cli:main
```

**Comment by ankitpokhrel** [comment]
The fix is available in the latest release [v1.1.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.1.0)
